examples/gallery/symbols/bars.py

```diff
@@ -0,0 +1,73 @@
+"""
```

Comment:
Perhaps adding the `r` prefix can fix the style failure.
```suggestion
r"""
```